app/about_us/page.js

```diff
@@ -45,18 +13,16 @@ export const Crew = () => {
         <h1>About us</h1>
         <section className="card">
           <h2>Our Values</h2>
-          <OurValues/>
+          <OurValues />
         </section>
         <section className="card">
           <h2>The crew</h2>
-          <OurCrew/>
+          <OurCrew />
         </section>
-
-         {/* TASK - React 1 week 1 */}
-         {/* Add in the "OurPartners" component here */}
+        <OurPartners />
```

Comment:
It seems like you are not following the same pattern for `<OurPartners />` as you are doing for the Our Values and Our Crew. It is best to stay consistent with these:
```
 <section className="card">
    <h2>The crew</h2>
     <OurCrew />
  </section>
   <section className="card">
    <h2>Our Partners</h2>
     <OurPartners />
  </section>
```

Reply:
Oh, thanks